interp/functions.go

```diff
@@ -320,11 +329,11 @@ func (p *interp) callUser(index int, args []Expr) (value, error) {
 	return null(), nil
 }
 
-// Call native-defined function with given name and arguments, return
+// Call native-defined function with given name and arguments,
```

Comment:
Thanks for catching. Let's make this "return its return value" like the comment on `callUser`.